local-runner/cf-local-container-launcher.yaml

```diff
@@ -28,26 +28,23 @@ Resources:
 
           def handler(event, context):
 
-            task = event.get('task', '').replace('"','\\"')
-
-            if task:
-              task = f'\"{task}\"'
-
             docker_command = ' '.join([
               "docker run --rm",
-              "__DOCKER_GATEWAY_HOST__",
               f"--name {event['name']}-{random_string(10)}", 
-              f"{'-d -p 6969:6969' if event['detached'] else ''}", 
-              f"--env SAMPLE_ID={event.get('sampleUuid', '')}",
-              event['image'],
-              task 
```

Comment:
So many simplifications evrywhere! 🤩
You'll also be able to do some cleanup in `app.py`

Reply:
just did so, thanks for the heads up!